jiant/huggingface_transformers_interface/modules.py

```diff
@@ -355,26 +354,83 @@ def forward(self, sent: Dict[str, torch.LongTensor], task_name: str = "") -> tor
         return self.prepare_output(lex_seq, hidden_states, input_mask)
 
     def get_pretrained_lm_head(self):
-        model_with_lm_head = pytorch_transformers.RobertaForMaskedLM.from_pretrained(
+        model_with_lm_head = transformers.RobertaForMaskedLM.from_pretrained(
             self.input_module, cache_dir=self.cache_dir
         )
         lm_head = model_with_lm_head.lm_head
         lm_head.predictions.decoder.weight = self.model.embeddings.word_embeddings.weight
         return nn.Sequential(lm_head, nn.LogSoftmax(dim=-1))
 
 
-class XLNetEmbedderModule(PytorchTransformersEmbedderModule):
+class AlbertEmbedderModule(HuggingfaceTransformersEmbedderModule):
+    """ Wrapper for ALBERT module to fit into jiant APIs.
+    Check HuggingfaceTransformersEmbedderModule for function definitions """
+
+    def __init__(self, args):
+        super(AlbertEmbedderModule, self).__init__(args)
+
+        self.model = transformers.AlbertModel.from_pretrained(
+            args.input_module, cache_dir=self.cache_dir, output_hidden_states=True
+        )
+        self.max_pos = self.model.config.max_position_embeddings
+
+        self.tokenizer = transformers.AlbertTokenizer.from_pretrained(
+            args.input_module, cache_dir=self.cache_dir
+        )  # TODO: Speed things up slightly by reusing the previously-loaded tokenizer.
+        self._sep_id = self.tokenizer.convert_tokens_to_ids("[SEP]")
+        self._cls_id = self.tokenizer.convert_tokens_to_ids("[CLS]")
+        self._pad_id = self.tokenizer.convert_tokens_to_ids("[PAD]")
+        self._unk_id = self.tokenizer.convert_tokens_to_ids("[UNK]")
```

Comment:
Double check these. I see Huggingface uses `<unk>` in a couple of places:
https://github.com/huggingface/transformers/pull/1683/files#diff-78dc6baf03f4d704be366101942aadd3R62

Reply:
Thanks, you are right. They have [SEP] [CLS] <pad> and <unk>.

Reply:
Do triple check that section of code. Errors like this could hurt performance in a subtle and hard-to-diagnose way across lots of examples.